app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/offline/OfflineDatabaseService.kt

```diff
@@ -234,17 +234,18 @@ class OfflineDatabaseService internal constructor(
     }
 
     override suspend fun updateUser(firebaseUser: FirebaseUser): OfflineUser {
-        userDao.update(
-            RoomUser(
-                firebaseUser.uid,
-                User.Type.FIREBASE,
-                firebaseUser.displayName,
-                firebaseUser.email
-            )
-        )
-        return fromUser(userDao.load(firebaseUser.uid, User.Type.FIREBASE)!!)
+        throw IllegalStateException("No Users Downloaded yet")
```

Comment:
If we intend to catch it (?) it should be a specific exception class, not a generic one.

Reply:
We do not intend to catch it for now I believe, it should be implemented in a later PR by @jonaskon  ?